app/views/roles/_form.html.erb

```diff
@@ -10,8 +10,14 @@
 
   <div class="tab-content">
     <div class="tab-pane active" id="primary">
+      <% if @role.cloned_from_id.present? && User.current.can?(:view_roles, @role.cloned_from) %>
```

Comment:
I consulted this with @rohoover already, the current PR status is the outcome. Note that with the recent `label_help` change the form looks more consistent since the notification is already converted to h5. I just need to do the rebase. I can update the screenshot after the rebase.